.rubocop_fixme.yml

```diff
@@ -23,7 +23,7 @@ Metrics/AbcSize:
 
 Metrics/ModuleLength:
   Exclude:
-    - 'app/controllers/concerns/hyrax/collections_controller_behavior.rb'
+    - 'app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb'
```

Comment:
See the discussion with @mjgiarlo in issue #1380.  I was originally going with admin, but he suggested dashboard because this functionality is not limited to admins.